src/test/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltrationIntegrationTest.java

```diff
@@ -0,0 +1,246 @@
+    public void testFilteringDPfromINFO() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --filterExpression 'DP < 8' --filterName lowDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testFilteringDPfromINFO.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testFilteringDPfromFORMAT() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " -genotypeFilterExpression 'DP < 8' --genotypeFilterName lowDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testFilteringDPfromFORMAT.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testInvertGenotypeFilterExpression() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --genotypeFilterExpression 'DP < 8' --genotypeFilterName highDP --invertGenotypeFilterExpression "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testInvertGenotypeFilterExpression.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testInvertJexlGenotypeFilterExpression() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --genotypeFilterExpression 'DP >= 8' --genotypeFilterName highDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testInvertJexlGenotypeFilterExpression.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testSetFilteredGtoNocall() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --genotypeFilterExpression 'DP < 8' --genotypeFilterName lowDP --setFilteredGtToNocall "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testSetFilteredGtoNocall.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+//    @Test //SELECT VARIANTS TEST
+//    public void testSetVcfFilteredGtoNocall() {
+//        String testfile = privateTestDir + "filteredSamples.vcf";
+//
+//        WalkerTestSpec spec = new WalkerTestSpec(
+//                "-T SelectVariants --setFilteredGtToNocall -R " + b37KGReference + " --variant " + testfile + " -o %s --no_cmdline_in_header",
+//                1,
+//                Arrays.asList("81b99386a64a8f2b857a7ef2bca5856e")
+//        );
+//
+//        spec.disableShadowBCF();
+//        executeTest("testSetVcfFilteredGtoNocall--" + testfile, spec);
+//    }
```

Comment:
Why is this commented-out `SelectVariants` test in `VariantFiltrationIntegrationTest`?